data/campaigns/Winds_of_Fate/scenarios/05x_Unexpected_Arrival.cfg

```diff
@@ -0,0 +1,329 @@
+        [foreach]
+            array=clashers
+            variable=this_unit
+            [do]
+                [unstore_unit]
+                    variable=this_unit
+                    x,y=recall,recall
+                [/unstore_unit]
+            [/do]
+        [/foreach]
+        {CLEAR_VARIABLE clashers}
+        [recall]
+            role=fighter_intendant
+            x,y=14,5
+        [/recall]
+        [recall]
+            role=clasher_intendant
+            x,y=13,6
+        [/recall]
+        {MODIFY_UNIT side=1 facing nw}
+        {MODIFY_UNIT side=2 facing se}
+        {PLACE_IMAGE (scenery/temple1.png) 09 15}
+        {PLACE_IMAGE (scenery/temple1.png) 10 17}
+        {PLACE_IMAGE (scenery/temple1.png) 11 13}
+        {PLACE_IMAGE scenery/whirlpool.png 20 1}
+    [/event]
+
+    [event]
+        name=start
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "Karron, you did not wait for my summons!"
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "You left no prospect of waiting.
+After your loud manifesto, the rogues-becoming began probing the Domain of Vladnir.
+Without your flight there would be not enough to hold against them.
+
+Vladnir counseled me here.
+So the youths you bestowed me I traded to other flights,
+for prime drakes willing to make the journey."
+        [/message]
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "Vladnir can not hold with <b>no</b> ally by his side."
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "Vladnir designed to follow me here. By now he is already faring."
+        [/message]
+        [message]
+            speaker=Reshan
+            image_pos=right
+            mirror=yes
+            message= _ "We have not yet the provisions for three flights within this domain.
+There could be hunger before the next moon!"
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "Gorlack we must--
+
+Gorlack, hatchlings do not belong at a counsel gathering."
+        [/message]
+        [message]
+            speaker=Arinexis
+            image_pos=right
+            mirror=yes
+            message= _ "We are <b>not</b> hatchlingss..."
+        [/message]
+        [message]
+            speaker=Arinexis
+            image_pos=right
+            mirror=yes
+            message= _ "Not hatchlings!"
+        [/message]
+        [message]
+            speaker=Zedrix
+            image_pos=right
+            mirror=yes
+            message= _ "Indeed, I am our pack'ss Elder. Ssuch longevity wandering starss bestow me. For over 11 yearsss have I lived."
+        [/message]
+        [message]
+            speaker=Reshan
+            image_pos=right
+            mirror=yes
+            message= _ "They are ‘Saurians’."
```

Comment:
Should be lower-case saurians (everywhere else in dialogue it is already lower-case).